common/lib/dependabot/pull_request_creator.rb

```diff
@@ -142,7 +144,8 @@ def azure_creator
         pr_description: message_builder.pr_message,
         pr_name: message_builder.pr_name,
         author_details: author_details,
-        labeler: labeler
+        labeler: labeler,
+        work_item: provider_metadata&.fetch(:work_item)
```

Comment:
Think this should default to nil otherwise it will raise

```suggestion
        work_item: provider_metadata&.fetch(:work_item, nil)
```